README.md

```diff
@@ -81,6 +81,8 @@ openssl pkcs12 -inkey key.pem -in certificate.crt -export -out certificate.p12
 You will be prompted to enter an export password, the again to verify.  Remember this password.
 
 ### Install the certificate on server
+These installation instructions are specific to Windows 7.  If your server is running on a different OS the process may differ.
```

Comment:
Might want to make the commit subject more verbose like "Updated README to clarify that cert install instructions are Windows 7 specific"